ldap/servers/slapd/disconnect_error_strings.h

```diff
@@ -27,6 +27,7 @@ ER2(SLAPD_DISCONNECT_BER_FLUSH, "Server failed to flush response back to Client
 ER2(SLAPD_DISCONNECT_IDLE_TIMEOUT, "Idle Timeout (nsslapd-idletimeout) - T1")
 ER2(SLAPD_DISCONNECT_REVENTS, "Poll revents - R1")
 ER2(SLAPD_DISCONNECT_IO_TIMEOUT, "IO Block Timeout (nsslapd-ioblocktimeout) - T2")
+ER2(SLAPD_DISCONNECT_PAGED_SEARCH_LIMIT, "Paged Search Limit Exceeded - T3")
```

Comment:
'Search Time Limit' would be more precise as there is also size limit that could be confusing

Reply:
Thats a good point, I will update